scipy/stats/_continuous_distns.py

```diff
@@ -362,6 +362,17 @@ def fit(self, data, **kwds):
 
         return loc, scale
 
+    def _munp(self, n):
+        """
+        @returns Moments of standard normal distribution for integer n >= 0
+
+        See https://arxiv.org/pdf/1209.4340.pdf
+        """
```

Comment:
Private methods usually do not need to have a docstring.  
A ref may be useful for a more esoteric distribution but normal distribution is usually not considered an esoteric distribution. 
For common formula, usually prefer to link to wikipedia, which in this case has this formula:
https://en.wikipedia.org/wiki/Normal_distribution#Moments
The arxiv paper has other useful formula too but they are not used in the `_munp`

Reply:
IMO arxiv reference is good. Wikipedia article is way easier to find tgen the arxiv preprint (can keep both if desired).

Only better link to the abstract, not pdf:
https://arxiv.org/abs/1209.4340

Reply:
@ev-br I'm ok with having the arxiv article but would be good to reference the formula which is being used here, e.g. eqn (16) from the latest version of the doc (v2) It also would be good to note the version number in case the author decided to submit an updated version which changes the formula numbers.

Reply:
I have added eq no and version no in arXiv link to landing page (rather than raw pdf)